tests/output/pjs/output_test.js

```diff
@@ -673,7 +673,82 @@ describe("Scratchpad Output Exec", function() {
         errors: []
     });
 
+    runTest({
+        title: "Modifying globals updates the draw loop",
+        code: function() {
+            var r = 100;
+            var c = color(255,0,0);
+            var draw = function() {
+                background(255);
+                fill(c);
+                ellipse(200, 200, r, r);
+            };
+        },
+        code2: function () {
+            var r = 200;
+            var c = color(255,255,0);
+            var draw = function() {
+                background(255);
+                fill(c);
+                ellipse(200, 200, r, r);
+            };
+        },
+        setup: function(output) {
+            var p = output.output.canvas;
+            sinon.stub(p, "fill");
+            sinon.stub(p, "ellipse");
```

Comment:
needed to do some custom setup to stub calls so that we can check what they were actually called with